### ISSUE TITLE
Crash report 7.0: NumberFormatException in EditPostActivity.onMediaUploadCancelClicked

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#5566]
```
Fatal Exception: java.lang.NumberFormatException: Invalid long: ""
       at java.lang.Long.invalidLong(Long.java:124)
       at java.lang.Long.parseLong(Long.java:345)
       at java.lang.Long.parseLong(Long.java:321)
       at java.lang.Long.valueOf(Long.java:511)
       at org.wordpress.android.ui.posts.EditPostActivity.onMediaUploadCancelClicked(EditPostActivity.java:2148)
       at org.wordpress.android.editor.EditorFragment$20.onClick(EditorFragment.java:1349)
       at android.support.v7.app.AlertController$ButtonHandler.handleMessage(AlertController.java:161)
       at android.os.Handler.dispatchMessage(Handler.java:111)
       at android.os.Looper.loop(Looper.java:207)
       at android.os.HandlerThread.run(HandlerThread.java:61)
```

**Comment by maxme** [comment]
Last log line before the crash was:
```
268	|	20:37:49:766	|	D/CrashlyticsCore d/WordPress-EDITOR: Image tapped, id=~url=file:///storage/emulated/0/WordPress/images/DSC_0272-2.JPG~meta={"align":"none","alt":"","attachment_id":"","caption":"","captionClassName":"","captionId":"","classes":"uploading","height":3008,"linkClassName":"","linkRel":"","linkTargetBlank":false,"linkUrl":"","size":"custom","src":"/storage/emulated/0/WordPress/images/DSC_0272-2.JPG","title":"","width":5344,"naturalWidth":5344,"naturalHeight":3008}~type=image
```
Notice the `id=~url=` (no value for `id`)

**Comment by mzorz** [comment]
Closed via #5573
